### PR TITLE
Adding docs for projections

### PR DESCRIPTION
**Pull request by plivesey**

### CODE REVIEW

File: docs/pages/055_projections.rst [hunk 1]
```diff
@@ -1,4 +1,86 @@
 Projections
 ===========
 
-TODO: Add docs here
+The Consistency Manager supports projections for models. This allows using the same ``modelIdentifier`` for multiple different classes. For instance, let's say you have two projections on the same base type - ``PersonModel`` and ``FullProfileModel``. Both models represent the same data (and share fields) so we want to use the same ID for both. However, this doesn't come for free. In general, it's not advised to use projections if you can avoid it and instead use composition (see the Alternatives section of this page).
```
Comment (dustin-shean-linkedin) [inline docs/pages/055_projections.rst]: two projections on => two projections of

Comment (freshnitesh) [inline docs/pages/055_projections.rst]: "This allows using the same `modelIdentifier` for multiple different classes"
It's clearer to say that we're using the same modelIdentifier for _instances_ of different classes.

File: docs/pages/160_FAQ.rst [hunk 2]
```diff
@@ -6,7 +6,7 @@ Immediately after setting new data, I'm getting multiple callbacks that a model
...
------------------------------------------------------------------------------------------
+If my models don't use projections, don't I need to return more data than I need?
+---------------------------------------------------------------------------------
 
-Possibly, but there are ways of modeling in ways which reduces total size and duplicated data. You can use composition to reduce the additional data. For instance, let's say you have a FullProfile model and a Person model. In most places, you only need a small part of the profile so you can use Person model which only has a few fields. Then, you can include the Person model in the FullProfile model. This keeps the guarantees about a model with the same ID having the same data, and means there is no duplicate data. Avoiding duplicate data is important if you want to only make one update per change (otherwise, you may need to update multiple models).
+Possibly, but there are ways of modeling in ways which reduces total size and duplicated data. You can use composition to reduce the additional data. See the Alternatives section of :doc:`055_projections` for more information.
```
Comment (freshnitesh) [inline docs/pages/160_FAQ.rst]: saying "ways" twice makes it confusing in my opinion

Consider: "Possibly, but there are ways of modeling in which total size and duplicated data are reduced."

**Comment by freshnitesh** [comment]
left a couple comments.

Overall, lgtm

**Pull request merged**